.github/workflows/build.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     uses: ./.github/workflows/_meta.yaml
     with:
       distro: 'debian'
-      codenames: '["buster", "bullseye", "bookworm"]'
```

Comment:
![image](https://github.com/jellyfin/jellyfin-ffmpeg/assets/14953024/e3ebc933-b4f0-420b-a8eb-8ed43990814e)

The oldoldstable/buster is reaching EOL. Also remove it from these scripts?
```
build
build.yaml
docker-build.sh
```